grc/core/params/param.py

```diff
@@ -8,6 +8,7 @@
 import ast
 import collections
 import textwrap
+import numbers
```

Comment:
I'm not quite sure of the GRC code style guidelines, but if in doubt, sort includes alphabetically?

Reply:
Good point, fixed now!